photonpump/connection.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self.connected = Event()
         self.disconnected = Event()
         self.stopped = Event()
-        self.ctrl_queue = ctrl_queue or asyncio.Queue(loop=self.loop)
```

Comment:
I think we can add some logic to still pass it for python < 3.10, but I'm not sure it makes sense since it's discouraged in previous versions too.

Reply:
the point of injecting a particular loop so we could have different photo pump components running on different loops, so for a more busy application, the loop which runs connection component could run on dedicated executor

Reply:
Can you give an example of such a case? I think it's generally advised to set a global current loop for each executor rather than passing it, i.e. in each executor we can do:
```py
new_loop = asyncio.new_event_loop()
asyncio.set_event_loop(new_loop)
```

Reply:
That being set, it might be worth adding a deprecation warning when explicitly passing the event loop to `photon-pump` and removing the `loop` parameter in future releases.

Reply:
@epoplavskis I've added a warning and made a bit of cleanup with all other usages of loop argument, I found some places where it was passed as an argument and never used. Going forward, we will need to get rid of passing the loop, and there are already existing and documented ways of supporting multi-threaded applications with "thread-local" loops.